CDS/scripts/js_utils/media-atom-lib.js

```diff
@@ -58,7 +58,13 @@ function fetchMetadata(connection) {
             .then(response => {
                 const title = response.data.title;
                 const description = response.data.description;
-                return Promise.all([datastore.set(connection, 'meta', 'atom_title', title), datastore.set(connection, 'meta', 'atom_description', description)])
+                const channelId = response.channelId;
```

Comment:
should this be `response.data.channelId`?

(I don't know the shape of the API response that well...)